domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ccj/RepaymentPlan.java

```diff
@@ -4,21 +4,16 @@
 import uk.gov.hmcts.cmc.domain.constraints.DateNotInThePast;
 import uk.gov.hmcts.cmc.domain.constraints.Money;
 
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.NotNull;
```

Comment:
Not sure why my intellij is auto arranging imports this way even when i move manually, i guess some setting !!

Reply:
you'll need to update your formatter, its on confluence but...
> The requested service is temporarily unavailable. It is either overloaded or under maintenance. Please try later.

Reply:
corrected